target/linux/pistachio/base-files/etc/uci-defaults/02_enable_bt

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+ln -s /etc/init.d/bt-csr /etc/rc.d/S22bt-csr
+sleep 2
+/etc/rc.d/S22bt-csr start
```

Comment:
The scripts from init.d does not get executed at startup but the files in the folder/etc/rc.d get executed, the files in this folder are symbolic links to the corresponding files in the init.d folder, to run the scripts in the init.d after boot we need to create the symbolic file by executing the command  /etc/init.d/bt-csr enable at least once, to avoid doing this manually we are creating the symbolic file in this script and starting it other wise again we will have to wait for the next boot for the script to be triggered automatically at boot